src/types.ts

```diff
@@ -1,26 +1,22 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
 import type {
   ComponentAnnotations as BaseComponentAnnotations,
   StoryAnnotations as BaseStoryAnnotations,
   StoryContext as BaseStoryContext,
   WebRenderer,
 } from 'storybook/internal/types';
 import type { Component, ComponentProps } from 'svelte';
-import type { SetOptional, Simplify } from 'type-fest';
```

Comment:
I think we can remove `type-fest` as `devDependency` now.

Reply:
[487c97c](https://github.com/storybookjs/addon-svelte-csf/pull/302/commits/487c97c47fb36c1c382b4ab1a875342d20ac566d)

Reply:
Never mind, I didn't notice it's being used in new stories `Types.stories.svelte`